### PR TITLE
warning modal implemented

### PR DESCRIPTION
**Pull request by tjw0612**
## Why

This pull request is needed because:

Added a new modal for double confirmation before generating results

image

## What

This pull request introduces the following:

 * `Changes introduced`

`Add Implementation details if needed`

### Screenshot

`Attach any screenshots here. Remove when unneeded`

## How to Test

`Give the reviewer(s) some clues on how to test your PR`

## Other

`This section is for things that can't be put into the previous sections. Remove when unneeded`

### CODE REVIEW

File: app/views/responses/show.html.erb [hunk 1]
```diff
@@ -21,9 +21,35 @@
...
+        <h5 class="modal-title" id="exampleModalCenterTitle">Generate Results</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        Are you sure that you want to end the poll prematurely and generate your restaurant result?
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
```
Comment (mxng) [inline app/views/responses/show.html.erb]: can you use btn-success or something so it's not grey? i might change the btn-secondary to orange instead if you prefer that

**Review by mxng — COMMENTED** [review]
btn-secondary change to btn-success or i can make btn-secondary have orange colour

File: app/views/responses/show.html.erb (hunk 1 above)
Comment (tjw0612) [reply app/views/responses/show.html.erb]: Updated to btn-success

**Review by TeeWhyKay — APPROVED** [review]
looks great!

**Pull request merged**